.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
Should pyproject.toml be also updated to reflect that Python 3.9 is the new minimum required version?

Reply:
good catch.